src/com/owncloud/android/datamodel/FileDataStorageManager.java

```diff
@@ -467,7 +467,7 @@ public void saveFolder(
     
 
     public boolean removeFile(OCFile file, boolean removeDBData, boolean removeLocalCopy) {
-        boolean success = true;
+        boolean success = false;
```

Comment:
I'm sorry to say this change breaks the condition in line 497 for input parameters ```file != null && !file.isFolder()``` ```removeDBData == false```. 

I'd say the easiest way to achieve what you want is with adding an ```else``` block in line 509.